src/styles/_base.scss

```diff
@@ -43,18 +43,18 @@ img {
 }
 
 button {
-  align-items: center;
-  background: $buttonLinkColor;
-  border-radius: 10px;
-  padding: 3vh;
-  transition-duration: 0.4s;
+  align-items: center !important;
```

Comment:
Stirtz usually yells at me for this, using !important is usually a sign that you should be a bit more specific in your styling targets. It can be difficult to track down styling bugs when there are a bunch of styles applied by a single target using !important.